clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -767,20 +767,20 @@ public void handle(HeartbeatResponse heartbeatResponse, RequestFuture<Void> futu
                 future.complete(null);
             } else if (error == Errors.COORDINATOR_NOT_AVAILABLE
                     || error == Errors.NOT_COORDINATOR) {
-                log.debug("Attempt to heartbeat since coordinator {} is either not started or not valid.",
+                log.info("Attempt to heartbeat since coordinator {} is either not started or not valid.",
```

Comment:
These changes are intentional: after trouble shooting several heartbeat issues I feel it is worth making them as INFO level logs than DEBUG, since 1) in normal production its frequency should be rare, and 2) once it happens, it would be very helpful determining the root causes. cc @hachikuji

Reply:
I think the only issue is that this message in particular might get a little spammy when we are discovering a new coordinator since it can take a little time for the cluster to converge. Seems fine to increase verbosity for the other errors though.

By the way, it looks like we're missing the word "failed" in the message